src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_raster/LocalServerDynamicWorkspaceRasterSample.java

```diff
@@ -0,0 +1,164 @@
+package com.esri.samples.localserver.local_server_dynamic_workspace_raster;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.List;
+
+import com.esri.arcgisruntime.layers.ArcGISMapImageLayer;
+import com.esri.arcgisruntime.layers.ArcGISMapImageSublayer;
+import com.esri.arcgisruntime.layers.RasterSublayerSource;
+import com.esri.arcgisruntime.localserver.DynamicWorkspace;
+import com.esri.arcgisruntime.localserver.LocalMapService;
+import com.esri.arcgisruntime.localserver.LocalServer;
+import com.esri.arcgisruntime.localserver.LocalServerStatus;
+import com.esri.arcgisruntime.localserver.RasterWorkspace;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.MapView;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.FileChooser;
+import javafx.stage.FileChooser.ExtensionFilter;
+import javafx.stage.Stage;
+
+public class LocalServerDynamicWorkspaceRasterSample extends Application {
+
+  private MapView mapView;
+  private static final LocalServer server = LocalServer.INSTANCE;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Local Server Dynamic Workspace Raster Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a control panel
+      VBox vBoxControl = new VBox(6);
+      vBoxControl.setMaxSize(240, 185);
+      vBoxControl.getStyleClass().add("panel-region");
+
+      // create the descriptive label
+      Label description = new Label("This application shows how to create a dynamic workspace connection " +
```

Comment:
Ok. Cool. I was going by what Mark had told me: Make the sample work exactly like it did at 10.2.4.
But I shouldn't have taken it so literally! Thanks for this.